src/payload.rs

```diff
@@ -1519,7 +1529,7 @@ impl RoaInfo {
 fn key_difference<K: Copy + Hash + Eq, V>(
```

Comment:
Nit/check: this should (also looking at the test below) return keys from `current` not occurring in `next`, but _not_ the other way around? Perhaps the comment describing the function should be more detailed, especially because the function name is generic (`_difference`)

Reply:
I have renamed the function to better reflect what it actually does. I also added a test for delta construction.